### PR TITLE
SAMZA-1779: Fix SamzaSql AvroRelConverter to handle BYTES and FIXED avro schema types

### PR DESCRIPTION
**Pull request by atoomula**

### CODE REVIEW

File: samza-sql/src/test/java/org/apache/samza/sql/system/TestAvroSystemFactory.java [hunk 1]
```diff
@@ -291,6 +292,7 @@ private Object createComplexRecord(int index) {
       GenericRecord record = new GenericData.Record(ComplexRecord.SCHEMA$);
       record.put("id", index);
       record.put("string_value", "Name" + index);
+      record.put("bytes_value", java.nio.ByteBuffer.wrap(("sample bytes").getBytes()));
```
Comment (vjagadish1989) [inline samza-sql/src/test/java/org/apache/samza/sql/system/TestAvroSystemFactory.java]: s/java.nio.ByteBuffer.wrap/ByteBuffer.wrap 

Or, Are there naming conflicts with ByteBuffer class that require us to fully namespace it?

**Review by vjagadish1989 — APPROVED** [review]
approved. just a question, otherwise looks good

File: samza-sql/src/test/java/org/apache/samza/sql/system/TestAvroSystemFactory.java (hunk 1 above)
Comment (atoomula) [reply samza-sql/src/test/java/org/apache/samza/sql/system/TestAvroSystemFactory.java]: Actually, nothing. I removed the namespace prefix.

**Comment by atoomula** [comment]
Another PR with https://github.com/apache/samza/pull/575/ has fixes for both byte and fixed types.

**Pull request closed without merging**